.circleci/config.yml

```diff
@@ -4,22 +4,18 @@ jobs:
   # Run Ansible-lint checks
   ansible-lint:
     docker:
-      - image: williamyeh/ansible:ubuntu16.04
+      - image: yokogawa/ansible-lint
```

Comment:
Update: due to upstream changes in `ansible-lint` version that lint check started failing too. Fixed that as well.